File: app/src/main/java/com/happypills/db/objects/pill.java
package com.happypills.db.objects;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "pills")
public class pill {

    @PrimaryKey(autoGenerate = true)
    private int uid;
    @ColumnInfo(name = "name")
    private String name;
    @ColumnInfo(name = "quantity")
    private int quantity;
    @ColumnInfo(name = "expiration")
    private long expiration;
    @ColumnInfo(name = "comment")
    private String comment;

    public int getUid() {
        return uid;
    }

    public void setUid(int uid) {
        this.uid = uid;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public long getExpiration() {
        return expiration;
    }

    public void setExpiration(long expiration) {
        this.expiration = expiration;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }
}

File: app/src/main/java/com/happypills/objects/Pill.kt
package com.happypills.objects

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "pillsTable")
class Pill {

    @PrimaryKey
    @ColumnInfo
    var title: String
    @ColumnInfo
    var quantity: Int
    @ColumnInfo
    var comment: String
    @ColumnInfo
    var expiryDate: String

    constructor(title: String, quantity: Int, comment: String, expiryDate: String) {
        this.title = title
        this.quantity = quantity
        this.comment = comment
        this.expiryDate = expiryDate
    }

}